### ISSUE TITLE
Add a page for updating/adding an ORCID

### ISSUE THREAD

**Issue by kimrutherford** [pombase/canto#1269]
We need a way to add an ORCID to an existing user.

**Comment by ValWood** [comment]
I'll remove "next" from this. We can wait a while. We also need to draft the text to tell them what we will use ORCIDs for...

**Comment by ValWood** [comment]
closing?

**Comment by kimrutherford** [comment]
We still don't have a good way to add ORCIDs to existing users.

**Comment by ValWood** [comment]
Usually you need to sign in and confirm your credentials, so I think you would see the ORCID field blank  if you hadn't added an ORCID?

**Comment by kimrutherford** [comment]
> Usually you need to sign in and confirm your credentials, so I think you would see the ORCID field blank if you hadn't added an ORCID?

Yep, but were to ask the community to add/update their ORCIDs we don't have a URL to give them to do that.  If we send them a new session then they could fill in their ORCID but otherwise we don't have a way to capture it.

**Comment by ValWood** [comment]
OK got it!

**Comment by kimrutherford** [comment]
I think we should consider collecting the ORCIDs via a Google Form (or something similar) and maybe combine the task with this:
  - pombase/curation#3517

**Comment by ValWood** [comment]
Good idea
